data/json/terrain.json

```diff
@@ -773,6 +815,7 @@
         "type" : "terrain",
         "id" : "t_utility_light",
         "name": "Utility Light",
+        "description": "An industrial flood light set up to illuminate the surroundings.  Smashing it doesn't seem like it'd produce any worthwhile salvage.",
```

Comment:
The latter part just seems like an issue with the item, not sure that should be noted here

Reply:
Again, it's mainly to alert the player it's not worth destroying for items (since it doesn't generate any). If the terrain gets changed, the description can always be altered.